config.json

```diff
@@ -387,52 +387,50 @@
         "name": "Bob",
         "uuid": "11ce2a73-3f43-4b14-b755-6c52dc71bdda",
         "practices": [
-          "strings"
+          "booleans"
```

Comment:
I'm not sure it really makes sense for me to check all these, so maybe just let me know how confident you are.